var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -26,6 +26,7 @@ class PyNumpy(PythonPackage):
         'numpy.distutils.command', 'numpy.distutils.fcompiler'
     ]
 
+    version('1.16.1', 'dafda51934f645d888866f98424521ae')
```

Comment:
Hi @svenevs https://github.com/spack/spack/pull/10738 added 1.16.2 which created a conflict here. Would you be able to address that as well as https://github.com/spack/spack/pull/10647#discussion_r259556762?